lively.morphic/morph.js

```diff
@@ -639,31 +635,31 @@ export class Morph {
         isStyleProp: true,
         type: 'Number',
         min: 0,
-        foldable: ['top', 'left', 'right', 'bottom'],
-        defaultValue: { top: 0, bottom: 0, right: 0, left: 0, valueOf: () => 0 },
+        foldable: ['topLeft', 'topRight', 'bottomRight', 'bottomLeft'],
+        defaultValue: { topLeft: 0, topRight: 0, bottomRight: 0, bottomLeft: 0, valueOf: () => 0 },
         get () {
           const v = this.getProperty('borderRadius');
-          return { ...v, valueOf: () => v.left };
+          return { ...v, valueOf: () => v.topLeft };
         },
         set (value) {
           if (!value) value = 0;
           if (obj.isNumber(value)) {
-            const left = value; const right = value; const top = value; const bottom = value;
-            value = { left, right, top, bottom };
+            const topLeft = value; const topRight = value; const bottomRight = value; const bottomLeft = value;
+            value = { topLeft, topRight, bottomRight, bottomLeft };
           }
           if (value.isRectangle) {
             value = {
-              left: value.left(),
-              right: value.right(),
-              top: value.top(),
-              bottom: value.bottom()
+              topLeft: value.x,
+              topRight: value.y,
+              bottomRight: value.width,
+              bottomLeft: value.height
             };
```

Comment:
Well this is not longer really supported I guess, so better just remove this one.